### DISCUSSION TITLE
Regarding this Repository

### DISCUSSION THREAD

**Discussion by Sathish36311** [stephanrauh/ngx-extended-pdf-viewer#2399] [Ideas]
Hi Guys, 
I'm curious to know the actual use case of this project, actual I'm also working on the same kind of project like extracting data from the pdf by rendering, it will be helpful when someone explain me about this.

**Comment by stephanrauh** [comment]
Well, it's simple: this is a PDF viewer, allowing Angular developers to use the PDF viewer of Firefox. There are also several other PDF viewers out there, but they only display the PDF without providing a UI.

What's probably a unique selling point is that ngx-extended-pdf-viewer also supports forms and a few editors modifying the annotation layer.

Regarding your project: ngx-extended-pdf-viewer allows you to extract text from the PDF. That support is fairly limited, but it seems to support many use-cases. See the live demo here: https://pdfviewer.net/extended-pdf-viewer/export-text
